portfolio/src/main/webapp/index.html

```diff
@@ -198,12 +198,22 @@ <h1 class ="titles">Photography</h1>
       </secion>
     </div>
     <div class="java-tests">
-      <form action="/data" method="POST">
+      <form action="/data" method="POST" name ="myForm">
         <p>Enter a comment</p>
         <textarea name="comment"></textarea>
         <input type="submit"/>
+        <br><br>
```

Comment:
It's usually frowned on to over-rely on the br tag (especially using more than one). This is usually an indication that you're doing something that's better accomplished with css. Plus, there are some usability concerns too (https://developer.mozilla.org/en-US/docs/Web/HTML/Element/br).